src/mono/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.Mono.cs

```diff
@@ -372,7 +371,7 @@ public virtual void BeginCatchBlock(Type exceptionType)
         public virtual void BeginExceptFilterBlock()
         {
             if (open_blocks == null)
-                open_blocks = new Stack(defaultExceptionStackSize);
+                open_blocks = new Int32Stack(defaultExceptionStackSize);
```

Comment:
It's interesting that L345 uses the null coalescing assignment operator and the rest of these don't.  Seems they could.

Reply:
Done. The `Int32Stack` is now created only when we begin a `try` block (in almost all other cases we used it only to check if it is empty).